src/Toppoki.purs

```diff
@@ -186,6 +186,22 @@ runPromiseAffE3 f a b c =  Promise.toAffE $ FU.runFn3 f a b c
 runPromiseAffE4 :: forall a b c d o. FU.Fn4 a b c d (Effect (Promise o)) -> a -> b -> c -> d -> Aff o
 runPromiseAffE4 f a b c d =  Promise.toAffE $ FU.runFn4 f a b c d
 
+-- | See [USKeyboardLayout](https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/lib/USKeyboardLayout.js) for a list of all key names.
+newtype KeyName = KeyName String
+
+-- | Shortcut for `keyboard.down` and `keyboard.up`.
+press
```

Comment:
should also link to https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#keyboardpresskey-options in the comment